source/velocity_boundary_conditions/gplates.cc

```diff
@@ -673,13 +673,13 @@ namespace aspect
                 first_data_file_number
                 + static_cast<unsigned int> (time_steps_since_start);
 
-              const bool load_both_files = std::abs(current_file_number - old_file_number) >= 1;
+              const bool load_both_files = std::abs(static_cast<int>(current_file_number - old_file_number)) >= 1;
```

Comment:
So ``current_file_number`` is an ``unsigned int`` and ``old_file_number`` is an ``int``. If you subtract them they are both converted to an unsigned first and might wrap around, right? I am not sure this is intended.